.env.sample

```diff
@@ -0,0 +1,26 @@
+# .env
+# Environment Configuration StarrScripts Sample 2024-04
+# Rename this file to .env and fill in the values accordingly.
+# Xseed
+## Download Client Names
+torrentclientname=""   # Example: "Qbit"
+usenetclientname=""    # Example: "SABnzbd"
+## Cross Seed API configuration
+xseed_host=""          # Example: "crossseed-server"
+xseed_port=""          # Example: "2468"
+## API Key for Cross Seed, if applicable
+xseed_apikey=""        # Example: "your-api-key"
+## Path to store the script's database of prior searches
+log_file=""            # Example: "/config/xseed_db.log"
```

Comment:
I'm not sure but IIRC you can do just `torrentclientname=` and it's the same as `""` in env and bash?

`undefined` or unset?